src/screens/volunteer/VolunteerRegistrationScreen.tsx

```diff
@@ -93,16 +91,7 @@ export const VolunteerRegistrationScreen = ({ navigation }: StackScreenProps<any
     <SafeAreaViewFlex>
       <DefaultKeyboardAvoidingView>
         <ScrollView keyboardShouldPersistTaps="handled">
-          <TitleContainer>
-            <Title
-              big
-              center
-              accessibilityLabel={`${texts.volunteer.registrationTitle} ${a11yLabel.heading}`}
-            >
-              {texts.volunteer.registrationTitle}
-            </Title>
-          </TitleContainer>
-          {device.platform === 'ios' && <TitleShadow />}
+          <SectionHeader title={texts.consul.registrationTitle} big center />
```

Comment:
Possibly not correct import, shouldn't it be `texts.volunteer.registrationTitle`?

Reply:
uh yes